src/SEOTools/SEOMeta.php

```diff
@@ -525,10 +561,10 @@ public function reset()
         $this->next = null;
         $this->prev = null;
         $this->canonical = null;
+        $this->amphtml = null;
         $this->metatags = [];
         $this->keywords = [];
         $this->alternateLanguages = [];
-        $this->canonical = null;
```

Comment:
why this has removed?

Reply:
Because it's duplicated a few lines above:

<img width="439" alt="screen shot 2018-02-11 at 23 59 21" src="https://user-images.githubusercontent.com/248183/36074733-92d04360-0f87-11e8-8605-23f6ad5aa55c.png">